src/content_script.js

```diff
@@ -329,7 +338,7 @@ function launchMultipleOnTC(callback) {
     });
 }
 
-
+console.log('staring injection of TC button');
 setEnv();
 setVendor();
 //initial load
```

Comment:
I might remove the console statements, or create a function that will only log in dev mode.